.storybook/preview.tsx

```diff
@@ -127,9 +116,32 @@ const withThemeSwitcher: Decorator = (
     );
 };
 
+const withAnnouncer: Decorator = (
+    Story,
+    {parameters: {addBodyClass}},
+) => {
+    // Allow stories to specify a CSS body class
+    if (addBodyClass) {
+        document.body.classList.add(addBodyClass);
+    }
+    React.useEffect(() => {
+        // initialize Announcer on load to render Live Regions earlier
+        initAnnouncer();
+        return () => {
+          if (addBodyClass) {
+            // Remove body class when changing stories
+            document.body.classList.remove(addBodyClass);
+          }
+        };
+      }, [addBodyClass]);
+    return (
+        <Story />
+    );
+};
+
 const preview: Preview = {
     parameters,
-    decorators: [withThemeSwitcher],
+    decorators: [withThemeSwitcher, withAnnouncer],
```

Comment:
Just a heads up that I also added some storybook decorators that are in `main` now so there might be a merge conflict when the feature branch is updated with `main`!